website/content/docs/connect/connect-internals.mdx

```diff
@@ -7,129 +7,68 @@ description: >-
-configuration to be set to specify a datacenter that is authoritative
-for intentions. In production setups with ACLs enabled, the
-[replication token](/consul/docs/agent/config/config-files#acl_tokens_replication) must also
-be set in the secondary datacenter server's configuration.
-
-## Certificate Authority Federation
-
-The primary datacenter also acts as the root Certificate Authority (CA) for Consul service mesh.
-The primary datacenter generates a trust-domain UUID and obtains a root certificate
-from the configured CA provider which defaults to the built-in one.
-
-Secondary datacenters fetch the root CA public key and trust-domain ID from the
-primary and generate their own key and Certificate Signing Request (CSR) for an
-intermediate CA certificate. This CSR is signed by the root in the primary
-datacenter and the certificate is returned. The secondary datacenter can now use
-this intermediate to sign new Consul service mesh certificates in the secondary datacenter
-without WAN communication. CA keys are never replicated between datacenters.
-
-The secondary maintains watches on the root CA certificate in the primary. If the
-CA root changes for any reason such as rotation or migration to a new CA, the
-secondary automatically generates new keys and has them signed by the primary
-datacenter's new root before initiating an automatic rotation of all issued
-certificates in use throughout the secondary datacenter. This makes CA root key
-rotation fully automatic and with zero downtime across multiple datacenters.
+Consul service mesh provides each service with an identity encoded as a TLS certificate. Consul uses this certificate to establish and accept connections to and from other services. The identity is encoded in the TLS certificate in compliance with the [SPIFFE X.509 Identity Document](https://github.com/spiffe/spiffe/blob/master/standards/X509-SVID.md). This enables Consul service mesh services to establish and accept connections with other SPIFFE-compliant systems.
+
+The client service verifies the destination service certificate against the [public CA bundle](/consul/api-docs/connect/ca#list-ca-root-certificates). In addition, the client provides its own client certificate to show its identity to the destination service. If the connection handshake succeeds, the connection is encrypted and authorized.
+
+The destination service verifies the client certificate against the [public CA bundle](/consul/api-docs/connect/ca#list-ca-root-certificates). After verifying the certificate, the next step depends upon the configured application protocol of the destination service. TCP (L4) services must authorize incoming _connections_ against the configured set of Consul [intentions](/consul/docs/connect/intentions), whereas HTTP (L7) services must authorize incoming _requests_ against those same intentions. If the intention check responds successfully, the proxy establishes a connection. Otherwise, the proxy rejects the connection.
```

Comment:
```suggestion
The destination service verifies the client certificate against the [public CA bundle](/consul/api-docs/connect/ca#list-ca-root-certificates). After verifying the certificate, the next step depends upon the configured application protocol of the destination service. TCP (L4) services must authorize incoming _connections_ against the configured set of Consul [intentions](/consul/docs/connect/intentions), whereas HTTP (L7) services must authorize incoming _requests_ against those same intentions. If the intention's authorization criteria is met, the proxy establishes a connection. Otherwise, the proxy rejects the connection.
```